dds_web/static/swagger.yaml

```diff
@@ -467,6 +467,379 @@ paths:
+                  type: string
+                  example: unituser1
+  /proj/busy/any:
+    get:
+      tags:
+        - superadmin
+      summary: Check if any project are busy CHECK METHOD
+      description: This method requires the data
+        to be passed in the request body instead of the query.
+        Since this does not comply with the openAPI standards, swagger cannot document it properly,
+        therefore we need to change/remove it in the future.
+      deprecated: true
+      operationId: anyProjectBusy
+      parameters:
+        - $ref: "#/components/parameters/defaultHeader"
+        - in: query
+          name: list
+          schema:
+            type: boolean
+          required: false
+          description: Check if user is listing the busy projects.
+            Otherwise just returns the number of busy projects.
+      responses:
+        "401":
+          $ref: "#/components/responses/UnauthorizedToken"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "500":
+          $ref: "#/components/responses/InternalServerlError"
+        "200":
+          description: succesful operation
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  num:
+                    type: integer
+                    example: 0
+  /stats:
+    get:
+      tags:
+        - superadmin
+      summary: Collect rows from reporting table and return them.
+      operationId: stats
+      parameters:
+        - $ref: "#/components/parameters/defaultHeader"
+      responses:
+        "401":
+          $ref: "#/components/responses/UnauthorizedToken"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "500":
+          $ref: "#/components/responses/InternalServerlError"
+        "200":
+          description: succesful operation
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  stats:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/Reporting"
+                  columns:
+                    type: array
+                    items:
+                      type: string
+                    example:
+                      [
+                        Active projects,
+                        Data Now (TB),
+                        Data Uploaded (TB),
+                        Date,
+                        Inactive projects,
+                        Project Owners,
+                        Researchers,
+                        Super Admins,
+                        TBHours Last Month,
+                        TBHours Total,
+                        Total projects,
+                        Total users,
+                        Unit Admins,
+                        Unit Personnel,
+                        Units,
+                      ]
```

Comment:
```suggestion
                      [
                        Active projects,
                        Data Now (TB),
                        Data Uploaded (TB),
                        Date,
                        Inactive projects,
                        Project Owners,
                        Researchers,
                        Super Admins,
                        TBHours Last Month,
                        TBHours Total,
                        Total projects,
                        Total users,
                        Unit Admins,
                        Unit Personnel,
                        Units,
                      ]
```
I suggest all such lists to be ordered in the way they appear in the methods definitions. That way it will easier to check

Reply:
Ahh, I see! 
I used this ordering because it is the order the request gives when run. Curious that flask somewhere reorders it. This is the image (needs to be run _flask stats_ first to have a reporting row, otherwise returns empty :s )

![image](https://github.com/ScilifelabDataCentre/dds_web/assets/46089290/7eef89f3-be95-40f9-b6a4-fa132bdb8c94)

Reply:
Yes, they are returned alphabetically sorted. I don't know, maybe we just leave it like that.